File: python/1929-concatenation-of-array.py
from typing import List

from typing import List, Optional
import collections
from utils import TreeNode, SLLNode as ListNode

class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans = []
        for i in range(2):
            for n in nums:
                ans.append(n)
        return ans

File: scala/1929-concatenation-of-array.scala
object Solution {
    def getConcatenation(nums: Array[Int]): Array[Int] = {
        val n = nums.length
        val ans = new Array[Int](2 * n)
        
        for (i <- 0 until n) {
            ans(i) = nums(i)
            ans(i + n) = nums(i)
        }
        
        ans
    }
}
